src/QsCompiler/Compiler/PluginInterface.cs

```diff
@@ -44,6 +44,12 @@ public static class RewriteStepPriorities
         /// evaluates classical computations as much as possible.
         /// </summary>
         public const int EvaluationOfClassicalComputations = 100;
+
```

Comment:
Unrelated to this change, but it would probably be nice to update the comment on this class to say which are higher priorities and which are lower. I.e. higher number - higher priority or otherwise.